examples/eth-pm/package.json

```diff
@@ -8,11 +8,10 @@
     "@ethersproject/providers": "5.7.0",
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
-    "@waku/core": "^0.0.16",
-    "@waku/create": "^0.0.12",
-    "@waku/interfaces": "^0.0.11",
-    "@waku/message-encryption": "^0.0.14",
-    "@waku/utils": "0.0.4",
+    "@waku/sdk": "^0.0.16",
+    "@waku/interfaces": "^0.0.15",
```

Comment:
`@waku/interfaces` aims to help maintainers (us) to maintain our APIs between the packages of the mono repo.

In terms of the types that the user may want for their own TypeScript library or implementation, should we re-export them from `@waku/sdk`?

Reply:
That's totally makes sense and I am going to add it soon with `@waku/relay` as well and some change to re-export of the `core`.

Reply:
https://github.com/waku-org/js-waku/pull/1409